sound/soc/sof/intel/mtl.c

```diff
@@ -641,6 +641,9 @@ int sof_mtl_ops_init(struct snd_sof_dev *sdev)
 	/* dsp core get/put */
 	/* TODO: add core_get and core_put */
 
+	sof_mtl_ops.hw_info |= SNDRV_PCM_INFO_HAS_LINK_ATIME |
+				SNDRV_PCM_INFO_HAS_WALL_CLOCK;
+
```

Comment:
commit message 

This patch is inspired by commit 9e94df3 ("ALSA: hda:
replace .wallclock by .get_time_info").

Reply:
thanks